Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	@rm -rf controller/controller
 
 build:
-	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit=$(COMMIT)" .
+	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit $(COMMIT)" .
```

Comment:
But, we got an error in "make image" with ```=```:
# go version
go version go1.4.2 linux/amd64

Do you mean I need to upgrade my golang?